src/dataEntities/startStop.ts

```diff
@@ -41,7 +49,7 @@ export abstract class StartStopService extends EventEmitter {
         // set started straight away to block the code below
         this.mStarting = true;
         await this.startInternal();
-        logger.info(`${this.name}: Started.`);
+        this.logger.info(`${this.name}: Started.`);
```

Comment:
You no longer need ${this.name} in any logs of this class, or of it's sub classes

Reply:
Removed.